transmart-batch/docs/data_formats/README.md

```diff
@@ -19,15 +23,15 @@
   * [subject-sample-mapping.md](subject-sample-mapping.md) - Tabular file structure for loading subject sample mappings for HD data.
 * mRNA gene expression data
   * [expression.md](expression.md) - Loading microarray gene expression data.
-  * *under development* - Loading readcounts and normalized readcounts data for mRNAseq and miRNAseq.
+  * *under development* - Loading readcounts and normalized readcounts data for transcripts and miRNA.
```

Comment:
@wardweistra I have now undone the changes in that line. 
I think transcript level RNA-Seq data should be a separate entry as it has its own parameter files and data/platform structure. But I have not added it yet, since it is not unlikely the platform format will change.